projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -255,6 +259,7 @@ public static function enqueue_scripts() {
 					'isAgencyAccount' => Jetpack_Manage::is_agency_account(),
 				),
 				'latestBoostSpeedScores' => $latest_score,
+				'protectStatus'          => $protect_status,
```

Comment:
The name protectStatus might be misleading since we have so much code related to statuses on the frontend and backend 🤔 Maybe it could be called `protectDetails` or something similar?

Reply:
Yeah, agreed..
How about `scanData`?, after all, it is the data retrieved from the `/scan` endpoint, and the data is all related to the scan of a site.